kernel/shop/customerorderview.php

```diff
@@ -16,6 +16,13 @@
 $tpl = eZTemplate::factory();
 
 $Email = urldecode( $Email );
+
+// workaround because it seems not possible to get an urlencoded "+" character accross $Params
+if ( $http->hasGetVariable( "email" ) )
+{
+    $Email = $http->getVariable( "email" );
+}
+
```

Comment:
I use a normal "get" parameter in orderlist.tpl, because "+" get through with them. I give this parameter a higher priority in the customerorderview.php. Does that seem wrong to you, @crevillo ?

Of course fixing the bug (?) with $Params would be the better solution.